x/evm/module.go

```diff
@@ -6,6 +6,8 @@ import (
 	"fmt"
 	"math/rand"
```

Comment:
where are we using rand?

Reply:
I think there is a function `RandomizeParams` which is used in the simulator.